### ISSUE TITLE
Data loader with multi outputs

### ISSUE THREAD

**Issue by jsalbert** [triagemd/keras-trainer#32]
Implement a dataloader with multi-outputs to use when there are multiple loss functions.
